docs/user-documentation/metadata.md

```diff
@@ -1,15 +1,17 @@
 # Metadata in Islandora 8
 
-> TL;DR: In Islandora 8 metadata values are stored in _fields_ attached to _entities_ (objects) which are then serialized as JSON-LD before being submitted to Fedora and/or indexed in a triple-store.
+> TL;DR: In Islandora 8, metadata is stored in _fields_ attached to _entities_ (nodes or media). Metadata values are serialized as RDF in the form of JSON-LD before being submitted to Fedora and/or indexed in a triplestore.
```

Comment:
I know that serialization into RDF is an important feature of Islandora 8, but I also feel that we as a community have been giving it a more dominant place in our discussions around metadata than I think it deserves. I say this because by making RDF serialization into JSON-LD that is then indexed in a triplestore so dominant a characteristic of the way that Islandora handles metadata, we're removing the focus from the simpler, more concrete idea that metadata in Islandora 8 is just plain old Drupal fields, and one very useful, but somewhat esoteric (let's be honest), feature of Islandora 8 is that the data in the fields can be expressed as RDF-in-JSON-LD-in-a-triplestore.

Reply:
I do think it deserves the more dominant place... as a repository much of our focus is getting our objects into a long-term persistence store (Fedora) and, currently, only the metadata that is serialized as JSON-LD gets that privilege.

I agree that the triple-store is of secondary importance (I only view it as an access point), but it is the JSON-LD to Fedora chain that earns it this place of importance.

Reply:
Good points, but I guess it depends on who the audience is. Does the average repo manager need to know about the plumbing that gives JSON-LD this privileged role and importance in Islandora, as long as everything just works? I'm asking that in as neutral a way as possible.

Reply:
I do think it is important to recognize this at a high-level that there is a difference between what is in your Drupal and what shows up in your Fedora. They don't necessarily need to know the plumbing (RDF-Mapping → JSON-LD Serializer → Normalize JSON-LD Array Alters) but the librarians here, at least, appreciate knowing there is a jump from Drupal to JSON-LD to Fedora.

Reply:
I'm not saying we shouldn't document that all that exists, I'm saying I think we've given those aspects of Islandora undue emphasis compared to the fact that the canonical version of the metadata, the one that most users interact with directly, is just plain old Drupal fields. It's just a matter of emphasis.

Reply:
Going back to the 'depends on who the audience is'; we have two sides of a coin here for perspective, the day-to-day 'access' side which is more interested in the Drupal fields with their editing and presentation and the 'preservation' side which is very concerned about what metadata is serialized. I think the overall documentation gives both concepts a good amount of coverage.

I think this particular line of documentation that spawned this discussion should also represent both sides of this coin. So, if we are to edit it, I suggest we _add_ a bit more emphasis to the 'Drupal fields is what we display to users' aspect, rather than take away from the 'Fields get serialized to JSON-LD for Fedora' aspect.

_Etymological side note:_

I don't know that I would call the Drupal fields 'canonical'. Certainly, the Drupal fields are what most users interact with directly, as you say, but I think the metadata as it is serialized to Fedora better fits the classical definition of 'canonical' where its role in preservation is a reflection of the term's history of reflecting a 'sacred' nature.

Reply:
The MIG has spent 2 years mapping to RDF, not to fields. I wanted the word "RDF" in here so there is a connection to all the predicates we've spent so long debating. 

Can I reword it to "Metadata values _can be_ serialized" rather than "_are_ serialized"?

Reply:
It's fine as is. I was simply making a point about emphasis.

Reply:
It was a good point to make, thank you!